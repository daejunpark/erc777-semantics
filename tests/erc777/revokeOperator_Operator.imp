test.preamble();

// Testing revokeOperator(Operator) when Operator is an operator of the caller
// should pass

test.authorizeOperator(1);
test.revokeOperator(1);
