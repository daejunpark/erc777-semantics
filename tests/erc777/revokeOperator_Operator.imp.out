7 is authorizing 1 as an operator
7 is revoking operator 1
